Gordon360/Services/AccountService.cs

```diff
@@ -196,6 +196,8 @@ public AccountViewModel GetAccountByUsername(string username)
                 accounts = accounts.Join(members, a => a.AD_Username, mv => mv.Username, (a, mv) => a).Distinct();
             }
 
+            // Don't show results for alumni who have requested their names not be shown
+            accounts = accounts.Where(a => a.ShareName != "N");
```

Comment:
I think it would make more sense to put this inside the `AccountsService.GetAccountsToSearch` method, in this block:
```csharp
IEnumerable<Alumni> alumni = Enumerable.Empty<Alumni>();
if (accountTypes.Contains("alumni"))
{
  alumni = _context.Alumni;
}
```
Since that method is the source of the `accounts` variable used here.

Reply:
Agreed!  Thanks for the suggestion.